kuma/javascript/src/section-link.jsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+import { render } from 'react-dom';
+
+import { gettext } from './l10n.js';
+
+import Anchor from './icons/anchor.svg';
+
+/**
+ * Returns a section anchor element with a nested SVG icon
+ * @param {Object} heading - The current heading DOM node
+ * @returns section anchor element as an HTML element
+ */
+export default function sectionAnchor(heading) {
+    const anchor = document.createElement('a');
+    anchor.href = `#${heading.id}`;
+    anchor.classList.add('section-link');
+    // heading.innerText is the current heading and will already be translated
+    anchor.setAttribute(
+        'aria-label',
+        `${gettext('Link to')} ${heading.innerText}`
+    );
```

Comment:
The reason that I didn't put a label in here is that Sheppy had filed a bug about the "Link to" text looking like part of the section title to some tools.   Presumably using aria-label instead of a hidden label element fixes that problem.